capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'Decorator'
```

Comment:
- [ ] 👍 You're on the right track, but there's a small issue. It seems there's no file with the exact name `Decorator`, but there is one similar to it named `decorator` in your codebase. To prevent any errors, please consider updating your code to match the correct file name, which should be `decorator`. This will ensure that your code runs smoothly without any issues. Keep up the good work! 👏📂